src/main/java/org/jenkinsci/plugins/codedx/model/StatisticGroup.java

```diff
@@ -3,45 +3,27 @@
 import java.util.HashSet;
 import java.util.Set;
 
-public enum StatisticGroup {
```

Comment:
Changed from `enum` to `class` so we can work with new finding status types without needing to update the plugin to explicitly recognize the status